### PR TITLE
remove subscribe to self

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: Basic-Video-Chat/Basic-Video-Chat/ViewController.m [hunk 1]
```diff
@@ -235,14 +230,7 @@ - (void)subscriber:(OTSubscriberKit*)subscriber
 - (void)publisher:(OTPublisherKit *)publisher
     streamCreated:(OTStream *)stream
 {
-    // Step 3b: (if YES == subscribeToSelf): Our own publisher is now visible to
```
Comment (robjperez) [inline Basic-Video-Chat/Basic-Video-Chat/ViewController.m]: Have you checked that this is not referenced in any tutorial or similar?

**Comment by robjperez** [comment]
small comment and LGTM

**Comment by msach22** [comment]
@robjperez removed from other samples

**Comment by robjperez** [comment]
:shipit:

**Pull request merged**